src/components/Products/Filters/Price.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+const Price = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const [minPrice, setMinPrice] = useState('');
+  const [maxPrice, setMaxPrice] = useState('');
+
+  const query = () => {
+    const locationSearch = location.search;
+    const urlParams = new URLSearchParams(locationSearch);
+    const minPriceParam = urlParams.get('minPrice');
+    const maxPriceParam = urlParams.get('maxPrice');
+
+    if (minPrice || maxPrice) {
```

Comment:
You can write it shorter. Something like:

useEffect(() => {
let searchLine = '';

if (+minPrice) {
searchLine.concat(`?minPrice=${minPrice}`)
}

if (+maxPrice) {
searchLine.concat(`${searchLine.length ? '?' : '&'}maxPrice=${maxPrice}`)
}

navigate(searchLine)
}, [minPrice, maxPrice])

Reply:
Fixed. Thanks!